zenithdb/zenith
# Zenith init from s3

Implements  #230 restore part.

This PR implements the initialization of pageserver from existing pgdata that is located on local fs or in s3 (minio).

To test it run
```
./target/debug/initdb -D /home/anastasia/zenith/pgdata_load
//Now start pgdata and create some tables
...
./target/debug/zenith init --snapshot-path=fs:/home/anastasia/zenith/pgdata_load
./target/debug/zenith start
./target/debug/zenith pg create main
./target/debug/zenith pg start main
//connect to pageserver and ensure that you see tables from local pgdata

```

Built on top of #284
